client/src/styles/styles.css

```diff
@@ -0,0 +1,4 @@
+/* body {
```

Comment:
Did you mean to leave this commented? This file seems unnecessary since it's not being used anywhere in the codebase.

Reply:
I left it incase we wanted to do any styling project wide. But im sure bootstrap will handle this. I will delete the file before re submitting a pull req